src/Craftwork/IdObfuscator/Exception/IdObfuscatorExceptionInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+declare(strict_types=1);
+
+namespace Craftwork\IdObfuscator\Exception;
+
+interface IdObfuscatorExceptionInterface
```

Comment:
Is there an Error/Exception interface this can extend? I've never seen this technique before and I'm not convinced it will work, as in I doubt you can try and catch an anything that isn't an Error/Exception.